rails-snow-queen/app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,69 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ */
+$theme-colors: (
+  "primary":#61D5DD,
+  "warning":#fa7f28,
+  "success":#76B041,
+  "info": #99C5E6,
+  "secondary": #FFC914,
+  "light": #F6F7F7,
+  "dark": #177E89
+);
+
+@import 'bootstrap';
+
+
+header {
+  background-color: rgba(97, 213, 221, 0.452);
```

Comment:
Should this color come from one of your variables?

Reply:
I will unify the colors and the variables once I start working on the look, thanks